cg/cli/workflow/balsamic/store.py

```diff
@@ -90,7 +83,7 @@ def generate_deliverables_file(context, dry, config_path, case_id):
         click.echo(click.style(f"Case {case_id} not found", fg="yellow"))
 
     if not config_path:
-        config_path = Path.joinpath(root_dir, case_id, case_id + ".json")
+        config_path = get_config_path(root_dir, case_id)
```

Comment:
Probably it is good also to check if config_file exists. Then `yaml.safe_load` downstream won't complain and we catch it early on.

Reply:
raising an exception here, how do you like that idea?

Reply:
Exception or make behave similar to previous code:
```
    if not case_obj:
        click.echo(click.style(f"Case {case_id} not found", fg="red"))
        context.abort()
```

Reply:
I'm affraid we will exit the invoking click-function as well if we do that?

Reply:
How about `raise click.Abort`?